README.md

```diff
@@ -49,10 +49,13 @@ If an url is included, then a placeholder should be included for the version. Ex
 * `github-branch-source:https://updates.jenkins.io/2.121/latest/github-branch-source.hpi` - will treat the url like the version, which is not likely the behavior you want
 * `github-branch-source::https://updates.jenkins.io/2.121/latest/github-branch-source.hpi` - will download plugin from url 
 
-If a plugin to be downloaded from the incrementals repository is requested using the -plugins option from the CLI, the plugin name should be enclosed in quotes, since the semi-colon can be interpretted as the end of the command.
+If a plugin to be downloaded from the incrementals repository is requested using the -plugins option from the CLI, the plugin name should be enclosed in quotes, since the semi-colon is otherwise interpretted as the end of the command.
 
 ```
-java -jar plugin-management-cli/target/plugin-management-cli-1.0-SNAPSHOT-jar-with-dependencies.jar -p "workflow-support:incrementals;org.jenkins-ci.plugins.workflow;2.19-rc289.d09828a05a74"
+java -jar plugin-management-cli/target/plugin-management-tool.jar -p "workflow-support:incrementals;org.jenkins-ci.plugins.workflow;2.19-rc289.d09828a05a74"
 ```
 
-For plugins listed in a .txt file, each plugin must be listed on a new line. Other [import formats](https://issues.jenkins-ci.org/browse/JENKINS-58147) coming soon. 
+For plugins listed in a .txt file, each plugin must be listed on a new line. Comments beginning with `#` will be filtered out. Other [import formats](https://issues.jenkins-ci.org/browse/JENKINS-58147) coming soon. 
+
+#### Other Information
+The plugin manager tries to use update center json to get the latest information about a plugin's dependencies. If this information is unavailable, it will use the dependency information in the provided in the MANIFEST.MF file after the plugin has been downloaded.
```

Comment:
This reads better to me:
```suggestion
The plugin manager tries to use update center json to get the latest information about a plugin's dependencies. If this information is unavailable, it will use the dependency information from the downloaded plugin's MANIFEST.MF file.
```

also not sure if json should be mentioned, the data format shouldn't really be relevant?
```suggestion
The plugin manager tries to use update center data to get the latest information about a plugin's dependencies. If this information is unavailable, it will use the dependency information from the downloaded plugin's MANIFEST.MF file.
```